### ISSUE TITLE
Will there be experiments w/ M5 data?

### ISSUE THREAD

**Issue by umitkaanusta** [thuml/Time-Series-Library#98]
Do you plan to experiment with TimesNet using M5 data? The winner is a LightGBM model and all but one solutions in the top5 have used GBDTs - it would be nice to see how TimesNet performs there

**Comment by elloza** [comment]
Good idea, I would like to propose something similar for an assignment for my students. 
E.g. Make a benchmark with the M5 dataset at least with some of the algorithms of this library.
Any recommendations for a newcomer to this library? Thank you very much!
